pro/src/components/ArchiveConfirmationModal/__specs__/utils.spec.ts

```diff
@@ -0,0 +1,120 @@
+import {
+  CollectiveBookingStatus,
+  CollectiveOfferStatus,
+  CollectiveOffersStockResponseModel,
+} from 'apiClient/v1'
+import { collectiveOfferFactory } from 'utils/collectiveApiFactories'
+
+import { canArchiveCollectiveOffer } from '../utils/canArchiveCollectiveOffer'
+
+describe('canArchiveCollectiveOffer', () => {
+  const stocks: Array<CollectiveOffersStockResponseModel> = [
+    {
+      beginningDatetime: String(new Date()),
+      hasBookingLimitDatetimePassed: false,
+      remainingQuantity: 1,
+    },
+  ]
+
+  const offerStatus = [
+    CollectiveOfferStatus.ACTIVE,
+    CollectiveOfferStatus.REJECTED,
+    CollectiveOfferStatus.INACTIVE,
+  ]
+
+  it.each(offerStatus)(
+    'should return true when collective offer can be archivable',
```

Comment:
Cool ! On pourrait carrément nommer le test 'should return true when collective offer status is %s' pour que le résultat du test affiche ceci :
````
✓ should return true when collective offer status is ACTIVE
✓ should return true when collective offer status is REJECTED
✓ should return true when collective offer status is INACTIVE
````
Plutôt que cela :
````
✓ should return true when collective offer can be archivable
✓ should return true when collective offer can be archivable
✓ should return true when collective offer can be archivable
````